cfg_samples/kanata.kbd

```diff
@@ -702,6 +702,36 @@ If you need help, please feel welcome to ask in the GitHub discussions.
   ;;
   ;; cm1 (cmd bash -c "echo hello world")
   ;; cm2 (cmd rm -fr /tmp/testing)
+
+  ;; One variant of `cmd` is `cmd-log`, which lets you control how
+  ;; running command, stdout, stderr, and execution failure are logged.
+  ;;
+  ;; The command takes two extra arguments at the beginning `<log_level>`,
+  ;; and `<error_log_level>`. `<log_level>` controls where the name
+  ;; of the command is logged, as well as the success message and command 
+  ;; stdout and stderr.
+  ;;
+  ;; `<error_log_level>` is only used if there is a failure executing the initial
+  ;; command. This can be if there is trouble spawning the command, or
+  ;; the command is not found. This means if you use `bach -c "thisisntacommand"`, as
```

Comment:
Should this be `bash` instead of `bach` or is the mispelling intended?

Reply:
Oop yep, that's supposed to be bash 😅

Reply:
Do you want me to squash or just add a small commit? I'll be able to do it this afternoon

Reply:
Either is fine 🙂